src/main/java/seedu/guest/model/guest/Email.java

```diff
@@ -48,7 +49,8 @@ public Email(String email) {
      * Returns if a given string is a valid email.
```

Comment:
Should this documentation be 'Returns true if...'?

Reply:
Good catch! Fixed